android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -883,6 +884,21 @@ fun String.toMediaType(): MediaType? = when (this) {
     else -> null
 }
 
+data class LockScreenControlConfig(
+    var isEnabled: Boolean = false
+)
+
+fun ReadableMap.toLockScreenControlConfig(): LockScreenControlConfig = LockScreenControlConfig().apply {
+    withBoolean("isEnabled") { isEnabled = it }
+}
+
+private val lockScreenConfigMap = mutableMapOf<PlayerConfig, LockScreenControlConfig>()
+var PlayerConfig.lockScreenControlConfig: LockScreenControlConfig
+    get() = lockScreenConfigMap[this] ?: LockScreenControlConfig()
+    set(value) {
+        lockScreenConfigMap[this] = value
+    }
+
```

Comment:
This config is a cross-platform wrapper, it does not exist in either native SDK.
It acts as a common name to reach the feature